src/main/java/com/commercetools/sync/commons/models/NonResolvedReferencesCustomObject.java

```diff
@@ -0,0 +1,56 @@
+package com.commercetools.sync.commons.models;
+
+import io.sphere.sdk.products.ProductDraft;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+
+public class NonResolvedReferencesCustomObject {
+
+    private String productKey;
```

Comment:
Why do we need this field?

Reply:
b18e91fd54a0dbf32f25970b1f82f0577d88eeb0